Classes/LinkHandler/AbstractLinkHandler.php

```diff
@@ -44,12 +44,14 @@
 use TYPO3\CMS\Recordlist\Tree\View\LinkParameterProviderInterface;
 
 /**
- * Link handler for page (and content) links
+ * Link handler for page (and content) links.
  */
+/** @codingStandardsIgnoreStart */
 abstract class AbstractLinkHandler extends Typo3LinkHandler implements LinkHandlerInterface, LinkParameterProviderInterface
```

Comment:
What does the CS check complain about here? It looks OK to me.

Reply:
Line exceeds 120 characters

Reply:
Can't you break the line, or does CS complain about that too?